controllers/v1/solutions.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * name : solutions.js
+ * author : Aman
+ * created-date : 03-Sep-2020
+ * Description : Solution related information.
+ */
+
+// Dependencies
+const solutionsHelper = require(MODULES_BASE_PATH + "/solutions/helper");
+
+module.exports = class Solutions extends Abstract {
+    
+    constructor() {
+        super(schemas["solutions"]);
+    }
+
+    /**
+   * @api {post} /kendra/api/v1/solutions/list
+   * Solutions lists.
+   * @apiVersion 0.0.1
+   * @apiName Solutions lists.
+   * @apiGroup Solutions
+   * @apiHeader {String} X-authenticated-user-token Authenticity token
+   * @apiParamExample {json} Request-Body:
+   * {
+    "query" : {
+        "externalId" : "EF-DCPCR-2018-001"
```

Comment:
@aman-tunerlabs - Keys should be limited to a certain set, not any query can be made, else some query on non index fields causes issues